motion-1/Overview.bs

```diff
@@ -163,19 +235,235 @@ Media: visual
 Animatable: yes
 </pre>
 
-A distance that describes the position along the specified <a>motion path</a>.
+Describes the position of the element along the specified <a>path</a>.
+
+<dl dfn-for="offset-distance" dfn-type="value">
+<dt><<length-percentage>></dt>
+<dd>The distance from the initial position of the shape or path to the position 
+of the element.
+
+Percentages are relative to the length of the path that is the distance between 
+the initial position and the end position of the path.</dd>
+
+<dt><<size>></dt>
+<dd>Decides a point used for measuring the distance from the initial position 
+	of the path.
```

Comment:
Yes, that's right.
The purpose of << size >> is to guarantee the constant distance between the element and the start point of the path when the offset on the path is given to the fixed << percetange >> and the changing << angle >> for defining the path.
For example, if there are elements like below,

 #circle1 {
    offset-path: 0deg;
    offset-distance: 50%;
  }
  #circle2 {
    offset-path: 45deg;
    offset-distance: 50%;
  }

if there isn't << size >>, #circle1 and #circle2 will have different offset from the start point of the path as << angle >> makes the path end at the edge of the containg block. This isn't an expected result for the use of << percentage >>, so it was resolved to add << size >> with << percentage >>.

I can see that definition about << size >> in here is not clear and it's hard to know the purpose of it.
Would this be clearer?

> << size >>
Decides the position of the end point of the path when it is used with << percentage >>.
When the offset of the element on the path is specified with << percentage >>, it needs to guarantee the constant culculated value regardless of << angle >>. To do that, << size >> gives a distance between the start point and the end point of the path.